airflow/www/views.py

```diff
@@ -1877,20 +1877,61 @@ def delete(self):
     @provide_session
     def trigger(self, session: Session = NEW_SESSION):
         """Triggers DAG Run."""
-        dag_id = request.values["dag_id"]
-        run_id = request.values.get("run_id", "")
+        dag_id = request.values.get("dag_id")
         origin = get_safe_url(request.values.get("origin"))
+        if not dag_id:
+            flash("No dag_id provided for trigger form.", "error")
```

Comment:
```suggestion
            flash("No dag_id provided.", "error")
```

Reply:
Yes, I also thought about a crash screen. But then I thought not adding complexity because the other routes when an invalid or broken DAG ID is given also do the same.
I added the validation here actually to ensure that no follow-up error appear and to validate early before any other data is processed so that we can be sure we have a string. (Follow-up code loads stuff from DB based on this and follow-up errors might be strange)